biz.aQute.repository/src/aQute/maven/provider/Releaser.java

```diff
@@ -58,8 +82,23 @@ public void close() throws IOException {
 				IO.store(localMetadata.toString(), metafile);
 
 				if (!localOnly) {
-					uploadAll(upload.iterator());
+					Iterator<Archive> iterator = upload.iterator();
+					uploadAll(iterator);
 					updateMetadata();
+					List<MavenBackingRepository> releaseRepositories = home.getReleaseRepositories();
+					String isSonatypePublish = context.getProperty("sonatypePublish", "false");
+					if (!releaseRepositories.isEmpty()
+						&& Boolean.parseBoolean(isSonatypePublish)) {
+						MavenBackingRepository mbr = releaseRepositories.get(0);
+						if (mbr.toString()
+							.contains("/sonatype_")) {
```

Comment:
Maybe something to improve in the future, as relying on `.toString()` might be fragile.